### ISSUE TITLE
Overlapping listr tasks (UI problem)

### ISSUE THREAD

**Issue by kirrg001** [TryGhost/Ghost-CLI#775]
I am getting overlapping listr tasks more often now. Looks like something is broken.

![lol](https://user-images.githubusercontent.com/1160712/43780909-0680d3c4-9a5c-11e8-98a9-c263e5f99847.png)

Version: latest (and 1.9)

**Comment by acburdine** [comment]
Looked into this a bit - looks like there's a bit of a race condition going on that's causing this.

Will work on it a bit more tomorrow.

**Comment by acburdine** [comment]
So the reason for this is a bit complicated, and (as far as I can tell) only occurs during the update command. One of the tasks in the update command (the "restarting Ghost" task) calls the start command, which in turn calls the doctor command. The doctor command runs a set of listr tasks, but because of how the command stack ends up being executed, those listr tasks end up instantiating a new `ui.renderer` instance, and thus you get the overlapping spinner.

There's a short-term and a long-term fix for this:

- Short term
Add a second argument to the start command run method that will optionally skip doctor checks. They really aren't needed during `update`, as most of the checks get run anyways before update even starts. What we could also do is make the `start` doctor checks part of the update listr stack, ensuring they still get run. This would fix the task list issue in the short term

- Long-term
Still need to write up another issue with some related things that need to happen first, will update this section once I've done that.
